### PR TITLE
reset ol, li markers to default setting

### PR DESCRIPTION
**Pull request by cstnaya**
The 3rd party theme tried to remove all default settings; while which is not we want.

( You can see this description in `theme/static/css/main.css` at line 323 - 331. )

So I reset `list-style-type` back to default.

in `theme/static/css/main.css` at line 361

```
ol,
ul {
  /* list-style: none; */
  margin: 0;
  padding: 0;
}
```

and also `theme/static/css/main.css` at line 657

```
li {
  /* list-style-position: outside; */
  /* list-style-type: disc; */
}
```

### CODE REVIEW

**Comment by yungyuc** [comment]
I wonder why this does not change the tailwind source? https://github.com/sciwork/swportal/blob/08562cc659f1f0c7901f4d73499fa8239296341c/tailwind/styles.css#L39

**Comment by cstnaya** [comment]
I found out that I can't just modify `main.css` file since it's the output file from tailwind-css... (´;ω;`)  

So I just made some change and this time I edit the `style.css` file.
And also I resolved the problem of `main.css` being too big... now it's just 17KB.

Hope this change can help you.

**Comment by yungyuc** [comment]
LGTM.  Thanks a lot.

File: tailwind/styles.css [hunk 1]
```diff
@@ -36,7 +36,13 @@ a { @apply text-red-600 underline; }
...
 ol, ul { padding-left: 1em; }
 
-li { @apply list-outside list-disc; }
+ol {
+  list-style: decimal;
+}
+
+ul {
+  list-style: disc;
+}
```
Comment (ezio-melotti) [inline tailwind/styles.css:45]: Since these two are already the default, it should be possible to just remove them altogether.

Comment (yungyuc) [reply tailwind/styles.css:45]: Thanks @ezio-melotti for the comment.  @cstnaya what do you think about making a new PR for it?

Comment (cstnaya) [reply tailwind/styles.css:45]: Within using `@tailwind base;`, lists will be set to unstyled, which made `ul` & `ol` markers are all invisible.
I want to overwrite this setting, and that's why I wrote down those lines.

Is it a better way if I replace the codes with lines below?
```
@layer base {
  ol, ul {
    list-style: revert;
  }
}
```

Comment (yungyuc) [reply tailwind/styles.css:45]: > Is it a better way if I replace the codes with lines below?

I am not familiar with tailwind enough to comment on that.  @chestercheng what do you think?

**Pull request merged**